### ISSUE TITLE
how to visual C3D features?

### ISSUE THREAD

**Issue by CCCCCVPR** [facebook/C3D#306]
hello Du Tran,i have a problem how can i realize to visualize deconvolutions of C3D learned feature map at special layers for example conv5b as the figures in your paper, i will appreciate you if you can  provide the source code ?

**Comment by Jillian2017** [comment]
I am also want the visualization code, I think it is a good way to verify the performance of model trained. Would you like to share it with us? Thank you very much.

**Comment by dutran** [comment]
I did not have a nice tool/code for it as I did that adhoc. e.g. I do forward pass on some videos then pick some layer activations that highest at that layers, take that gradient and set the gradients of other filters in the same layer to be zeros. Then backprop only that gradient to the image space (here is video as we have time) for visualization, to see which part of the image/video cause the activation of the filter.
